### DISCUSSION TITLE
F.A.Q.s , Q&A , Tricks using TB Launcher

### DISCUSSION THREAD

**Discussion by victor0d** [TBog/TBLauncher#290] [General]
This discussion is intended to collect different information, tricks and usage experiences that can help users resolving tasks or configurations that may not be instantly obvious

**Comment by victor0d** [comment]
**LOCK PHONE GESTURE with ACCESSIBILITY PERMISSIONS**

* PROBLEM
  Locking the phone with gestures, while being able to unlock it with finger prints or face unlocking (rather than lock it down).

* SOLUTION
  Set up the feature 'Run/Launch' shortcut within the behaviour gestures, and assign an application or shortcut that carry out the functionality.

There are many apps that can provide this functionality. Nevertheless, just to suggest here a simple and efficient one, "Lock", available in F-droid and Github (https://github.com/seguri/lock). It doesn't require root, but obviously, it will require accessibility access.

**Reply by ltguillaume** [reply]
Seguri's Lock didn't work for me, at least not on Android 14. Its accessibility service closed or crashed often and/or when triggered, an overlay on the screen was visible, while the screen did not turn off.

I just found https://github.com/BLumia/pineapple-lock-screen (F-Droid: https://f-droid.org/packages/net.blumia.pineapple.lockscreen.oss/, Google Play: https://play.google.com/store/apps/details?id=net.blumia.pineapple.lockscreen) and it does work reliably.

**Comment by TBog** [comment]
### Capture a bug report from device
Report the logcat when a simple description is not enough.

### [Official steps](https://developer.android.com/studio/debug/bug-report#bugreportdevice)
To get a bug report directly from your device, do the following:

1. Be sure you have [Developer Options](https://developer.android.com/studio/debug/dev-options) enabled.
2. In Developer options, tap Take bug report.
3. Select the type of bug report you want and tap Report.
4. After a moment you get a notification that the bug report is ready (see figure 2).

To share the bug report, tap the notification.

### Alternative
* From 2016 - haven't tested if this still works - https://android.gadgethacks.com/how-to/android-basics-capture-logcat-for-detailed-bug-reports-0167774/
  * video https://www.youtube.com/watch?v=W6iDw76UpQw
* From 2021 - [Capturing The Bug Report Through ADB Command.](https://techblogs.42gears.com/generate-bug-report-on-android-devices/) - see point 3
